openreview/venue/invitation.py

```diff
@@ -1680,3 +1680,153 @@ def build_expertise_selection(committee_id):
         if self.venue.use_senior_area_chairs:
             build_expertise_selection(self.venue.get_senior_area_chairs_id())
 
+    def set_registration_invitations(self):
+
+        venue = self.venue
+        venue_id = self.venue_id
+
+        for registration_stage in venue.registration_stages:
+            committee_id = registration_stage.committee_id
+
+        readers = [venue_id, committee_id]
```

Comment:
shouldn't all this be indented, so it's all in the for loop?

Reply:
yeah! I just fixed it